pkgs/by-name/oc/oci2git/package.nix

```
@@ -0,0 +1,38 @@
+{
+  fetchFromGitHub,
+  lib,
+  openssl,
+  pkg-config,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "oci2git";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "Virviil";
+    repo = "oci2git";
+    rev = "7f384808fb729fc57cd69a7b10dd8a2c4680729f";
```

why aren't we using the tag here?
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "oci2git";
  version = "0.1.3";

  src = fetchFromGitHub {
    owner = "Virviil";
    repo = "oci2git";
    tag = "v${finalAttrs.version}";
```